libs/common/src/lib/label/label-field/label-field.component.ts

```diff
@@ -25,15 +25,24 @@ export class LabelFieldComponent {
   @Input() set kind(kind: LabelSetKind) {
     this._kind.next(kind);
   }
+  private _labelSets = new ReplaySubject<LabelSets>();
+  @Input() set labelSets(value: LabelSets | undefined) {
+    if (value) {
+      this._labelSets.next(value);
+    }
+  }
   @Input() size: Size | undefined;
   @Input() disabled: boolean = false;
 
   @Output() selectionChange = new EventEmitter<Classification[]>();
   @Output() hasLabels = new EventEmitter<boolean>();
 
-  labelSets$ = this._kind.pipe(
-    switchMap((kind) =>
-      kind === LabelSetKind.RESOURCES ? this.labelsService.resourceLabelSets : this.labelsService.paragraphLabelSets,
+  labelSets$ = merge(
+    this._labelSets,
+    this._kind.pipe(
+      switchMap((kind) =>
+        kind === LabelSetKind.RESOURCES ? this.labelsService.resourceLabelSets : this.labelsService.paragraphLabelSets,
+      ),
```

Comment:
I don't understand well why you do a merge here… if you provide the labelSets as an input, do you still need to get the labelsets by kind from `labelsService`?

Reply:
`kind` and `labelSets` inputs are exclusive, only one of the should be provided.
Would you replace it by something like this ?
```
  ngOnInit() {
    if (labelsets input is provided) {
      this.labelsets$ = this._labelSets...
    }
    else {
      this.labelSets$ = this._kind...
    }
  }
```

Reply:
OK, I think the fact those inputs are exclusive should be documented in the code then (a simple comment above the inputs would work)

Reply:
Fixed